types/block.go

```diff
@@ -528,11 +530,55 @@ func NewCommit(blockID BlockID, precommits []*CommitSig) *Commit {
 	}
 }
 
+// Construct a VoteSet from the Commit and validator set. Panics
+// if precommits from the commit can't be added to the voteset.
+func CommitToVoteSet(chainID string, commit *Commit, vals *ValidatorSet) *VoteSet {
```

Comment:
These methods probably deserve tests to not decrease our test-coverage (?)

Reply:
Yes you're right.